tests/test_cli.py

```diff
@@ -107,3 +108,79 @@ def test_create_user_config():
         "foo": None,
         "main": 3,
     }
+
+
+class MockResponse:  # pylint: disable=too-few-public-methods
+    def __init__(self, data, status_code):
+        self.data = data
+        self.status_code = status_code
+
+    def json(self):
+        return self.data
```

Comment:
Is this class used at all? The below uses only `.json()`, so basically passes back the original dict.

Reply:
Yeah, old code. :+1: